CONTRIBUTING.md

```diff
@@ -170,6 +170,16 @@ for coding advice). The code must adhere to the following robustness principles
 are important for software that runs autonomously and continuously without direct
 interaction with a human (such as this Collector).
 
+### Recommended Libraries / Defaults
+
+In order to simplifiy developing within the project, library recommendations have been set
+and should be followed.
```

Comment:
Add also something like, "There are places in the codebase that are still not following this, but new code should strive to align with the recommendations."

Reply:
👍🏽  ,

Yeah, I agree with that.